docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -28,7 +28,7 @@ zkApp programmability is not yet available on the Mina Mainnet. You can get star
 
 In previous tutorials, you learned how to deploy and run transactions on a local network. 
 
-In this tutorial, you use the `zk config` command to create a deploy alias, request tMINA funds to pay for transaction fees, and deploy a project to a live network.
+In this tutorial, you use the `zk config` command to create a deploy alias, request MINA funds to pay for transaction fees, and deploy a project to a live network.
```

Comment:
In this instance, isn't the faucet giving test MINA? the tMina should be accurate

Reply:
Yes, faucet does gives the test mina.
I tried to check in wordlist and didn't found `tMINA` and also when I requested test MINA and can see them as MINA in my wallet. Attaching screenshot for reference. Hence I came to conclusion that it was not supposed.

![Screenshot 2024-01-23 at 11 24 38 PM](https://github.com/o1-labs/docs2/assets/55076843/41a0bee9-0dfb-458d-8299-feaf82970633)

But now that I re-read the tutorial again I think I made mistake. In the sense of tutorial tMINA is correct.
I will close this PR. Thanks for helping 😄

Thank You 😃